e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -290,18 +293,18 @@ describe("scenarios > search", () => {
       it("should hydrate created_by filter", () => {
         cy.intercept("GET", "/api/user").as("getUsers");
 
-        cy.request("GET", "/api/user/current").then(
-          ({ body: { common_name, id } }) => {
-            cy.visit(`/search?q=orders&created_by=${id}`);
+        cy.visit(`/search?q=orders&created_by=${NORMAL_USER_ID}`);
 
-            cy.wait("@search");
-            cy.wait("@getUsers");
+        cy.wait("@search");
 
-            cy.findByTestId("created_by-search-filter").within(() => {
-              cy.findByText(common_name).should("exist");
-              cy.findByLabelText("close icon").should("exist");
-            });
-          },
+        cy.findByTestId("created_by-search-filter").within(() => {
+          cy.findByText("Robert Tableton").should("exist");
+          cy.findByLabelText("close icon").should("exist");
+        });
+
+        cy.findByTestId("search-result-item-name").should(
```

Comment:
e2e tests are failing here: https://github.com/metabase/metabase/actions/runs/6381889343/job/17319996663?pr=34214

It's either a bug or a flake that should be fixed.

Reply:
fixed